src/main/resources/view/DarkTheme.css

```diff
@@ -350,3 +350,6 @@
     -fx-background-radius: 2;
     -fx-font-size: 11;
 }
+
```

Comment:
Excessive lines here prehaps?

Reply:
There needs to be a new line an end of file if not the checkstyle will fail